features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/services/login/introspect.jag

```diff
@@ -24,8 +24,8 @@
 
     var userInfoEndpoint = MGT_TRANSPORT + HOST + ":" + getServerPort() + "/oauth2/userinfo";
     var introspectEndpoint = MGT_TRANSPORT + HOST + ":" + getServerPort() + "/oauth2/introspect";
-    var tokenP1 = request.getCookie("WSO2_AM_TOKEN_1_Default").value;
-    var tokenP2 = request.getCookie("AM_ACC_TOKEN_DEFAULT_P2").value;
+    var tokenP1 = request.getCookie("WSO2_AM_TOKEN_1_Default") ? request.getCookie("WSO2_AM_TOKEN_1_Default").value : null;
```

Comment:
I have applied this diff locally and I'm still getting the exception in the carbon log.
I think we need to handle the null pointer in `/publisher/services/refresh/refresh.jag`

Reply:
Please refer : https://github.com/wso2/product-apim/issues/5655